kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md

```diff
@@ -183,81 +183,74 @@ Manu II. 176 and Yāj. I. 99, 100, 102 show that _Devapūjā_ is different from[
 
-77. For 38 उपचारs vide व्रतराज p. 44. 36 उपचारs are enumerated in एकादशीतत्त्व p. 58 quoting from आह्निकचिन्तामणि. They are not set out here for reasons of space. The 16 उपचारs quoted from प्रपञ्चसार by व.क्रि. कौ. P 156 and ए. त. p. 59 are : आसनं स्वागतं पाद्यमय॑माचमनीयकम् । मधुपर्काचमस्नान घसनाभरणानि च। सुगन्धसुमनोधूपदीपनैवेद्यवन्दनम् । प्रयोजयेदर्चनायामुपचारांश्च षोडश । अर्घ्यपाद्याचमनकमधुपर्काचमा अपि । गन्धादयो निवेद्यान्ता उपचारा दश क्रमात् । गन्धादयो निवेद्यान्ताः पूजा पञ्चोपचारिकी। उपचारैर्यथाशक्ति देवतामन्वहं यजेत् ।. The व. कि. को. (p. 157) proceeds : 'अत्यन्ताशक्तावुपचारद्वयेनापि पूजामाह कालिकापुराणे । ... अभावे गन्धपुष्पाभ्यां तदभावे च भक्तितः।'. The व. क्रि. को. p 156 notes that there is slight difference in the order in which some apacāras are offered '
+[^77]: For 38 उपचारs vide व्रतराज p. 44. 36 उपचारs are enumerated in एकादशीतत्त्व p. 58 quoting from आह्निकचिन्तामणि. They are not set out here for reasons of space. The 16 उपचारs quoted from प्रपञ्चसार by व.क्रि. कौ. P 156 and ए. त. p. 59 are : आसनं स्वागतं पाद्यमध्य॑माचमनीयकम् । मधुपर्काचमस्नान वसनाभरणानि च। सुगन्धसुमनोधूपदीपनैवेद्यवन्दनम् । प्रयोजयेदर्चनायामुपचारांश्च षोडश । अर्घ्यपाद्याचमनकमधुपर्काचमा अपि । गन्धादयो निवेद्यान्ता <U>उपचारा दश क्रमात्</U> । गन्धादयो निवेद्यान्ताः <U>पूजा पञ्चोपचारिकी</U>। उपचारैर्यथाशक्ति देवतामन्वहं यजेत् ।. The व.क्रि. को. (p. 157) proceeds : 'अत्यन्ताशक्तावुपचारद्वयेनापि पूजामाह कालिकापुराणे । ... अभावे <u>गन्धपुष्पाभ्यां तदभावे च भक्तितः</u>।'. The व. क्रि. को. p 156 notes that there is slight difference in the order in which some apacāras are offered 'अत्र अर्ध्यानन्तरं पाद्यमाचमनीयं चोक्तम् । शारदातिलकादिषु पाद्याचमनीयानन्तरमध्यर्मुक्तम् । कालिकापुराणे पाद्यार्थ्याचमनीयक्रम उक्तः ': Vide ब्रह्मवैवर्त (ब्रह्मखण्ड) chap. 26.90-92 for उपचारs.
```

Comment:
पाद्यमर्घ्यमाचमनीयकम्

Reply:
corrected.

Reply:
[^77]: For 38 उपचारs vide व्रतराज p. 44. 36 उपचारs are enumerated in एकादशीतत्त्व p. 58 quoting from आह्निकचिन्तामणि. They are not set out here for reasons of space. The 16 उपचारs quoted from प्रपञ्चसार by व.क्रि. कौ. P 156 and ए. त. p. 59 are : आसनं स्वागतं पाद्यमर्घ्यमाचमनीयकम् । मधुपर्काचमस्नान वसनाभरणानि च। सुगन्धसुमनोधूपदीपनैवेद्यवन्दनम् । प्रयोजयेदर्चनायामुपचारांश्च षोडश । अर्घ्यपाद्याचमनकमधुपर्काचमा अपि । गन्धादयो निवेद्यान्ता <U>उपचारा दश क्रमात्</U> । गन्धादयो निवेद्यान्ताः <U>पूजा पञ्चोपचारिकी</U>। उपचारैर्यथाशक्ति देवतामन्वहं यजेत् ।. The व.क्रि. को. (p. 157) proceeds : 'अत्यन्ताशक्तावुपचारद्वयेनापि पूजामाह कालिकापुराणे । ... अभावे <u>गन्धपुष्पाभ्यां तदभावे च भक्तितः</u>।'. The व. क्रि. को. p 156 notes that there is slight difference in the order in which some apacāras are offered 'अत्र अर्ध्यानन्तरं पाद्यमाचमनीयं चोक्तम् । शारदातिलकादिषु पाद्याचमनीयानन्तरमध्यर्मुक्तम् । कालिकापुराणे पाद्यार्थ्याचमनीयक्रम उक्तः ': Vide ब्रह्मवैवर्त (ब्रह्मखण्ड) chap. 26.90-92 for उपचारs.